app/views/courses/about.html.erb

```diff
@@ -83,4 +104,49 @@
       </p>
     <% end %>
   </div>
+
+  <div class="govuk-grid-column-one-third">
+    <aside class="related">
+      <h3 class="govuk-heading-m">Copy content from another course</h3>
+      <p class="govuk-body">Fill in this page with content from another course.</p>
+      <p class="govuk-body">This will replace any existing content.</p>
+      <%= form_for course, url: about_provider_course_path(course.provider.provider_code, course.course_code), method: :get do |f| %>
+        <div class="govuk-form-group">
+          <label class="govuk-label govuk-label--s" for="copy-from">Copy from</label>
+          <select id="copy-from" name="copy_from" class="govuk-select">
+            <option value="" selected="selected">Pick a course</option>
```

Comment:
will this get displayed when there is no courses to copy from?

Reply:
Done